README.rst

```diff
@@ -54,6 +64,12 @@ You can use `pip <https://pypi.org/project/pip/>`_ to install PyAnsys Geometry.
 
     pip install ansys-geometry-core
 
+You can also install PyAnsys Geometry from `Conda-Forge <https://anaconda.org/conda-forge/ansys-geometry-core>`_:
```

Comment:
This references Conda-Forge. The URL takes me to the account login page for Anaconda. Later installation sections reference Conda. I'm not sure if we are talking about all the same thing here. If so, using different names for the same thing is confusing.

Reply:
Spot on! This link is not working because https://github.com/conda-forge/staged-recipes/pull/26585 hasn't been merged yet. The moment it is merged, it will trigger the creation of the package. I just wanted to stay ahead and leave the PR set up. Thanks for reviewing as always @PipKat!